py/picca/delta_extraction/data.py

```diff
@@ -37,6 +37,7 @@
     "rejection log file",
     "save format",
     "num processors",
+    "delta extraction single exposure",
```

Comment:
not so sure this name is really clear, potentially should add something to the parameter descriptions in the readme-files or have a more self-explanatory name. Maybe something like "spectral coadding method", but given that this is only applied for cont fitting might be misleading... @corentinravoux @iprafols any ideas?

Reply:
I think that as long as we explain this somewhere this name should be fine